### ISSUE TITLE
Is there a plan to upgrade the UltimateALPR-SDK to support Jetpack 4.6?

### ISSUE THREAD

**Issue by mochechan** [DoubangoTelecom/ultimateALPR-SDK#205]
The [document](https://github.com/DoubangoTelecom/ultimateALPR-SDK/blob/master/Jetson.md) shows that "The SDK was tested using JetPack 4.4.1, the latest version from NVIDIA and we will not provide support for any other version."

Currently, the latest Jetpack 4.6 was released. Is there a plan to upgrade the UltimateALPR-SDK to support Jetpack 4.6?

**Comment by DoubangoTelecom** [comment]
Yes, without ETA

**Comment by davidfungf** [comment]
When I execute "sudo chmod +x ./prepare.sh && sudo ./prepare.sh" on Jetson Nano (JetPack 4.6), it returned the following errors.

```
*[COMPV INFO]: [CompVBase] [Base] modules initialized
*[COMPV INFO]: [UltAlprSdkTRT] Optimizing models in [../../../assets/models.tensorrt/] folder...
*[COMPV INFO]: [FileUtils] Loading files in ../../../assets/models.tensorrt/ ...
***[COMPV ERROR]: function: "open()"
file: "/home/ultimate/compv/base/compv_sharedlib.cxx"
line: "70"
message: [CompVSharedLib] Failed to load library with path=/home/safeuser/ultimateALPR-SDK-master/binaries/jetson/aarch64/libultimatePluginTensorRT.so, Error: libnvinfer.so.7: cannot open shared object file: No such file or directory
***[COMPV ERROR]: function: "open()"
file: "/home/ultimate/compv/base/compv_sharedlib.cxx"
line: "72"
message: Operation Failed (COMPV_ERROR_CODE_E_NOT_FOUND) ->
***[COMPV ERROR]: function: "newObj()"
file: "/home/ultimate/compv/base/compv_sharedlib.cxx"

```

**Comment by DoubangoTelecom** [comment]
We require JetPack 4.4.1. https://github.com/DoubangoTelecom/ultimateALPR-SDK/blob/master/Jetson.md#getting-started_requirements

**Comment by DoubangoTelecom** [comment]
https://groups.google.com/g/doubango-ai/c/zrtba7PlAHs/m/JraJtXjtBAAJ?utm_medium=email&utm_source=footer

**Comment by scrocker-amp** [comment]
Is there any update on this?  I would like to use this on a custom Jetson carrier board but so far, only JetPack 4.5 and 4.6 have been tested with the board.

**Comment by garciatruan** [comment]
Hi guys, I don't want to sound reckless, but it's almost a year since this problem is open, we know it's without ETA, but it sounds like a reasonable time, doesn't it? we really appreciate your effort on this! Thanks a lot!

**Comment by DoubangoTelecom** [comment]
Fixed by v3.11.0 with support for Jetpack 5.1.0 afc97140e9784b6e2478e5abf2249346b26111b2
